src/Sylius/Bundle/ChannelBundle/spec/Context/FakeChannel/FakeChannelPersisterSpec.php

```diff
@@ -72,15 +72,7 @@ function it_persists_fake_channel_codes_in_a_cookie(
         $response->headers = $responseHeaderBag;
         $responseHeaderBag
             ->setCookie(Argument::that(function (Cookie $cookie) {
-                if ($cookie->getName() !== '_channel_code') {
-                    return false;
-                }
-
-                if ($cookie->getValue() !== 'fake_channel_code') {
-                    return false;
-                }
-
-                return true;
+                return $cookie->getName() === '_channel_code' && $cookie->getValue() == 'fake_channel_code';
```

Comment:
We missed one `=` here 
```suggestion
                return $cookie->getName() === '_channel_code' && $cookie->getValue() === 'fake_channel_code';
```